tests/js/PrettierLintBearTest.py

```diff
@@ -0,0 +1,19 @@
+from bears.js.PrettierLintBear import PrettierLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file = """foo(
+  reallyLongArg(),
+  omgSoManyParameters(),
+  IShouldRefactorThis(),
+  isThereSeriouslyAnotherOne()
+);"""
+
+# Start ignoring PycodestyleBear, LineLengthBear
+bad_file = """
+foo(reallyLongArg(), omgSoManyParameters(), IShouldRefactorThis(), isThereSeriouslyAnotherOne());
+"""
+# Stop ignoring
+
+PrettierLintBear = verify_local_bear(PrettierLintBear,
+                                     valid_files=(good_file,),
+                                     invalid_files=(bad_file,))
```

Comment:
Okay for a start, but optimally I would like to have more tests ;) (can be done in a later issue)